supertokens/querier.go

```diff
@@ -198,7 +198,7 @@ func (q *Querier) SendPutRequest(path NormalisedURLPath, data map[string]interfa
 type httpRequestFunction func(url string) (*http.Response, error)
 
 func (q *Querier) sendRequestHelper(path NormalisedURLPath, httpRequest httpRequestFunction,
-	numberOfTries int) (map[string]interface{}, error) {
+	numberOfTries int) (map[string]string, error) {
```

Comment:
Why did you change all this to `string`? Maybe the JSON response contains a number?